Formula/v8.rb

```diff
@@ -18,11 +18,22 @@ class V8 < Formula
     sha256 cellar: :any, mojave:        "e624c7696f994d9f538da54be2f3ef3b78ceda13f420e78bd86b6bde196c82e7"
   end
 
-  depends_on "llvm" => :build
   depends_on "ninja" => :build
   depends_on "python@3.9" => :build
 
-  depends_on xcode: ["10.0", :build] # required by v8
+  on_macos do
+    depends_on "llvm" => :build
+    depends_on xcode: ["10.0", :build] # required by v8
+  end
+
+  on_linux do
+    depends_on "pkg-config" => :build
+    depends_on "gcc"
+    depends_on "glib"
+    depends_on "icu4c"
```

Comment:
Building against Homebrew ICU is a pain -- can we keep using the bundled one as on macOS?

Reply:
Using the bundled ICU definitely helped to simplify the Linux changes!